lib/beaker-pe/install/pe_utils.rb

```diff
@@ -101,6 +101,28 @@ def install_ca_cert_on(host, opts)
           end
         end
 
+        # Return agent nodes with 'lb_connect' role
+        def loadbalancer_connecting_agents
+          lb_connect_nodes = select_hosts(roles: ['lb_connect'])
+          lb_connect_agents = lb_connect_nodes.reject { |h| h['roles'].include?('loadbalancer')}
+        end
+
+        # Returns true if the host is an agents that is configured  to connect to loadbalancer
+        #
+        # @param [Host] host to be validated
+        def is_lb_connect_agent(host)
+          host['roles'].include?('lb_connect')  && ! host['roles'].include?('loadbalancer')
```

Comment:
2 things about this method:

1. This method should conform to Ruby conventions: boolean method names should end with a `?` character
2. It looks like this method re-implements the functionality in `loadbalancer_connecting_agents`. A better way to implement this, so that future changes to this functionality wouldn't need to be made in two places, would be to implement this method using the results of the other like so:
```ruby
def is_lb_connect_agent?(host)
  loadbalancer_connecting_agents.include?(host)
end
```